pkg/internal/logging/metadata/level.go

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package metadata
+
+// Level defines all available log levels for logging messages.
+type Level int
+
+// Log levels.
```

Comment:
Check if you still need it

Reply:
Check to see if I still need what?

Reply:
// note: below constants are copy of 'log.Level' constants added to avoid circular references,
// care should be taken before changing below constants including their order.
// For any updates in level constants below, corresponding changes has to be done for constants in 'log.Level'.

Reply:
oh my bad - I missed that. I'll take a look

Reply:
yep - it's still needed. Trying to remove it creates a cycle, just like in Aries-Framework-Go